www/userguide/userguide.md

```diff
@@ -0,0 +1,126 @@
+\page userguide User Guide
+\tableofcontents
+
+**CodeChecker** is a static analysis infrastructure built on the [LLVM/Clang
+Static Analyzer](http://clang-analyzer.llvm.org) toolchain, replacing
+[`scan-build`](http://clang-analyzer.llvm.org/scan-build.html) in a Linux or
+macOS (OS X) development environment.
+
+\section list_of_runs List of runs
```

Comment:
Maybe we should write about these concepts. It may not be completely intuitive what a product or run (update mode, etc.) is. However I think the description of these should go to another user guide, because if the user is already on the web interface then he or she is done with storing results to a run.